contrib/kafka/pkg/adapter/adapter.go

```diff
@@ -77,8 +77,9 @@ func (a *Adapter) ConsumeClaim(sess sarama.ConsumerGroupSession, claim sarama.Co
 	logger := logging.FromContext(context.TODO())
 
 	for msg := range claim.Messages() {
-		logger.Info("Received: ", zap.Any("value", string(msg.Value)))
-
+		logger.Info("Received: ", zap.String("topic", string(msg.Topic)),
+			zap.Int("partition", int(msg.Partition)),
+			zap.Int("offset:", int(msg.Offset)))
```

Comment:
minor:
`offset:` ends with a `:`  - should have the same for `topic` and `partition` ?

Reply:
Ah, nice catch.  Thanks for the review, I've pushed a commit that fixes this and updates the README accordingly!

Reply:
These should be `topic`, `partition`, and `offset`. No need for the `:` separator - zap's formatter will take care of that.